src/components/ImportPage.tsx

```diff
@@ -1,39 +1,36 @@
 import * as React from 'react';
 import Helmet from 'react-helmet';
-import { match as RouteMatch } from 'react-router-dom';
+import { useRouteMatch } from 'react-router-dom';
 import { PageSection, Title } from '@patternfly/react-core';
 import { QueryClient, QueryClientProvider } from 'react-query';
 import { ImportWizard } from './ImportWizard/ImportWizard';
 import { NamespaceContext } from 'src/context/NamespaceContext';
 
 const queryClient = new QueryClient();
 
-interface PipelineWizardPageProps {
-  match: RouteMatch<{ namespace: string }>;
-}
-
-const ImportPage: React.FunctionComponent<PipelineWizardPageProps> = ({
-  match: {
+const ImportPage: React.FunctionComponent = () => {
+  const {
     params: { namespace },
-  },
-}) => (
-  <>
-    <Helmet>
-      <title>Crane</title>
-    </Helmet>
-    <QueryClientProvider client={queryClient}>
-      <NamespaceContext.Provider value={namespace}>
-        <>
-          <PageSection variant="light">
-            <Title headingLevel="h1">Import application</Title>
-          </PageSection>
-          <PageSection variant="light" type="wizard">
-            <ImportWizard />
-          </PageSection>
-        </>
-      </NamespaceContext.Provider>
-    </QueryClientProvider>
-  </>
-);
+  } = useRouteMatch<{ namespace: string }>();
```

Comment:
This little refactor is an artifact of the reverted changes for the `from` URL param, but I left it this way because it's still an improvement (`useRouteMatch` is less awkward than using the `RouteMatch` prop type)